package.json

```diff
@@ -2,7 +2,7 @@
   "name": "proposal-flatmap",
   "version": "1.0.0",
   "description": "",
-  "main": "ecmarkup.js",
+  "main": "polyfill.js",
```

Comment:
You could use the `es-abstract` package and get things like ToObject and ToLength, etc - but proposal repo polyfills aren't meant to be published (and thus it probably shouldn't be in the "main").

(realistically, proposal repos should always omit "main")